db/mockdata.py

```diff
@@ -0,0 +1,183 @@
+"""
+This module contains classes required by the compact API.
+"""
+
+import os
+import binascii
+from datetime import datetime, timedelta
+import db
+from sqlalchemy import text
+
+
+# Token expiration time in minutes
+TOKEN_EXPIRATION_TIME = 60
+
+
+class User(object):
+    def __init__(self, row):
+        serial, created, name, auth_token = row
+        self.id = serial
+        self.name = name
+        self.created = created
+        self.api_key = auth_token
+
+    @staticmethod
+    def get_id(mb_id):
+        with db.engine.connect() as connection:
+            result = connection.execute(text(""" SELECT id FROM "user" WHERE
+                                            musicbrainz_id = :mb_id """), {"mb_id": mb_id})
+            row = result.fetchone()
+            if row:
+                return row[0]
+            return None
+
+    @staticmethod
+    def load_by_name(mb_id):
+        with db.engine.connect() as connection:
+            result = connection.execute(text(""" SELECT * FROM "user" WHERE
+                                            musicbrainz_id = :mb_id """), {"mb_id": mb_id})
+            row = result.fetchone()
+            if row:
+                return User(row)
+            return None
+
+    @staticmethod
+    def load_by_id(serial):
+        with db.engine.connect() as connection:
+            result = connection.execute(text(""" SELECT * FROM "user" WHERE id=:id """), {"id": serial})
+            row = result.fetchone()
+            if row:
+                return User(row)
+            return None
+
+    @staticmethod
+    def load_by_apikey(api_key):
+        with db.engine.connect() as connection:
+            result = connection.execute(text(""" SELECT * FROM "user" WHERE
+                                            auth_token = :auth_token """), {"auth_token": api_key})
+            row = result.fetchone()
+            if row:
+                return User(row)
+            return None
+
+    @staticmethod
+    def get_play_count(user_id):
+        """ Get playcount from the given user name.
+        """
+        with db.engine.connect() as connection:
+            result = connection.execute(text(""" SELECT COUNT(*) FROM listen WHERE
+                                            user_id = :user_id """), {"user_id": user_id})
+            return int(result.fetchone()[0])
+
+class Session(object):
+    def __init__(self, row):
+        serial, userid, sid, api_key, timestamp = row
+        self.id = serial
+        self.user = User.load_by_id(userid)
+        self.sid = sid
+        self.api_key = api_key
+        self.timestamp = timestamp
+
+    @staticmethod
+    def load(session, api_key=None):
+        """ Load the session details from the database.
+            If the api_key is also supplied then verify it as well.
+        """
+        dic = {'sid': session}
```

Comment:
dic is a bit of a generic name. Please use something a little more specific for the task.